### ISSUE TITLE
Missing country names that aren't recognised in any locale

### ISSUE THREAD

**Issue by UsAndRufus** [countries/countries#670]
I've been using this gem to to migrate 10,000 country names in many different languages to ISO 3166 alpha 2 codes. It goes through every locale that `countries` supports and calls `ISO3166::Country.find_country_by_name` to find the country. 

This worked for ~97% of the country names, but the following were missing. I haven't had chance to check them all, but it may be helpful to maintainers. I would write a PR myself but there are 23 open and I'm timing out on this task. 

For the record, all of these locations were AFAIK imported from the [Google geocoding API](https://developers.google.com/maps/documentation/javascript/geocoding), so they are standard names. A lot are fairly obvious - e.g. Danish is missing the Danish word for Denmark. Some are variant names like The Gambia rather than Gambia.

```
Danmark
Slovenija
Tyskland
澳洲
Storbritannien
Ελλάδα
Türkiye
Polska
美国
Hrvatska
Espanha
대한민국
Ελβετία
Кипр
台灣
Suomi
Sverige
ประเทศไทย
Česká republika
미국
Magyarország
Velká Británie
新西兰
美國
المملكة المتحدة
Norge
Stati Uniti
Francja
인도
Καναδάς
Espanya
Германия
USA
Regno Unito
Spojené kráľovstvo
Ηνωμένο Βασίλειο
Бразилия
Велика Британија
Ισπανία
ישראל
Germania
英国
新加坡
Соединенные Штаты Америки
Macedonia (FYROM)
Islanti
馬來西亞
영국
България
Kosovo
Молдова
Paesi Bassi
中國
Nouvelle-Zélande
Česko
Сполучені Штати Америки
加拿大
Svizzera
Швейцария
Нидерланды
Таиланд
Великобританія
Países Baixos
Švýcarsko
캐나다
Латвия
Нидерландия
Srbija
Eesti
Slovaška
Frankrike
ארצות הברית
Япония
일본
Pilipinas
越南
Wielka Brytania
荷蘭
Съединени щати
The Bahamas
ایران
Myanmar (Burma)
Казахстан
Dania
İngiltere
澳大利亚
Tschechien
Ungheria
The Gambia
ไลบีเรีย
```

**Comment by pmor** [comment]
Thanks for the list!

`ISO3166::Country.find_country_by_name` currently does not search the list of translated names, but that will be addressed as part of #71.

Most of these are working in #717, the remaining ones are translations that do not match the translations we get from https://salsa.debian.org/iso-codes-team/iso-codes

leaving them here for now, I'll address them after #717

```
澳洲
台灣
ประเทศไทย
Česká republika
Velká Británie
美國
المملكة المتحدة
Соединенные Штаты Америки
Macedonia (FYROM)
馬來西亞
Kosovo
中國
Сполучені Штати Америки
Великобританія
Нидерландия
荷蘭
The Bahamas
Myanmar (Burma)
İngiltere
The Gambia
```

**Comment by UsAndRufus** [comment]
Very welcome! Glad it was helpful, I figured it was useful given the initial dataset was so large.
